packages/status-react/src/protocol/use-messages.tsx

```diff
@@ -91,20 +91,20 @@ export const useMessages = (channelId: string): Result => {
   // const [state, dispatch] = useReducer<Result>((state,action) => {}, {})
 
   const [data, setData] = useState<any[]>(() =>
-    client.community.getMessages(channelId)
+    client.community.chats.get(channelId).getMessages()
```

Comment:
Notice: changes chat references here and elsewhere in the ui. However, working with already enumerated chats directly instead of accessing them with id via `community` might be better, if possible.

Reply:
Consider sending messages via community's method.

Reply:
Or via `chat.sendTextMessage()` if chat instances would be passed to components?